daprdocs/content/en/operations/configuration/preview-features.md

```diff
@@ -13,7 +13,8 @@ Currently, preview features are enabled on a per application basis when running
 
 ### Current preview features
 Below is a list of existing preview features:
-- [Actor Reentrancy]({{<ref actor-reentrancy.md>}})
+- [Actor reentrancy]({{<ref actor-reentrancy.md>}})
+- [Actor type metadata with reminders partitioning]({{<ref actor-reminders-partitions.md>}})
```

Comment:
Per #1592 this list should be under support and not this "how to" page

Reply:
Again, are we addressing #1592 in this PR as well?

Reply:
Yes. Remove the link from here. I am getting Hal to remove these and the list of preview features should just be in one Support document, in one place.

Reply:
Done.